larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -2420,39 +2406,19 @@ else if(attribute.isEmpty()){
 	 * @param propertyName The name of the ignore we are checking, in the example 'ignoreContentBetweenKeys'
 	*/
 	public static ArrayList<String> findAttributesForIgnore(String propertyName) {
-		ArrayList<String> attributes = null;
-
-		switch (propertyName) {
-			case "decodeUnzipContentBetweenKeys":
-			  	attributes = new ArrayList<>( Arrays.asList("key1", "key2", "replaceNewlines") );
-			  	break;
-			case "canonicaliseFilePathContentBetweenKeys":
-			case "replaceRegularExpressionKeys":
-			case "ignoreContentBetweenKeys":
-			case "ignoreKeysAndContentBetweenKeys":
-			case "removeKeysAndContentBetweenKeys":
-			case "replaceKey":
-			case "formatDecimalContentBetweenKeys":
-			case "replaceEverywhereKey":
-				attributes = new ArrayList<>( Arrays.asList("key1", "key2") );
-				break;
-			case "ignoreRegularExpressionKey":
-			case "removeRegularExpressionKey":
-			case "ignoreContentBeforeKey":
-			case "ignoreContentAfterKey":
-				attributes = new ArrayList<>(List.of("key"));
-				break;
-			case "ignoreCurrentTimeBetweenKeys":
-				attributes = new ArrayList<>( Arrays.asList("key1", "key2", "pattern", "margin", "errorMessageOnRemainingString") );
-				break;
-			case "ignoreKey":
-			case "removeKey":
+		return switch (propertyName) {
+			case "decodeUnzipContentBetweenKeys" -> new ArrayList<>(Arrays.asList("key1", "key2", "replaceNewlines"));
+			case "canonicaliseFilePathContentBetweenKeys", "replaceRegularExpressionKeys", "ignoreContentBetweenKeys", "ignoreKeysAndContentBetweenKeys",
+				 "removeKeysAndContentBetweenKeys", "replaceKey", "formatDecimalContentBetweenKeys", "replaceEverywhereKey" ->
+					new ArrayList<>(Arrays.asList("key1", "key2"));
+			case "ignoreRegularExpressionKey", "removeRegularExpressionKey", "ignoreContentBeforeKey", "ignoreContentAfterKey" ->
+					new ArrayList<>(List.of("key"));
```

Comment:
Moet dit echt een `ArrayList<>` zijn? Of mag het "gewoon" een `List<>` zijn die de method teruggeeft? Dan hoef je niet iedere keer `new ArrayList<>(...)` te doen.

Reply:
Ja inderdaad. Heb het aangepast. Dit was bestaande code wat ik effe had omgezet naar enhanced switch. Ik neem dit gelijk effe mee.